### ISSUE TITLE
Cant get past "Come closer" upon enslavement

### ISSUE THREAD

**Issue by beepboop6** [SkyrimLL/SDPlus#1090]
Title says it all pretty much. Running this in AE. when someone enslaves me, it gives the popup saying "your owner wants to inspect you" and a dialogue box saying to "come closer, i want to see what you got" or something along those lines. I cant seem to progress past this. Going closer/right up to the owner doesnt trigger anything

**Comment by thrush91214** [comment]
SAME, was wondering if that was all there was to it at first, **surely** there had to be more right???
Anyone know resolution?

**Comment by maxxaukai-ops** [comment]
I am also having this issue on SE, can't seem to find a solution to it.

Edit:
Okay, I found the most elegant solution I can.
I am playing with a fairly complex modlist so first I double checked requirements, make sure to follow the mod page for that I used the LoversLab page.
Once I knew that wasn't the issue, I was stumped on cause and don't have more to offer on that.
It's by no means a perfect solution, but what I did to fix it was use the console command sqo _SD_enslavement to find the Your Master wants to inspect you objective which was 61, and then used the command setobjectivecomplete _SD_enslavement 61 1 which FINALLY ticked off the inspection objective, this stopped it from flagging as failed for me when time passed and I didn't lose any standing with the owner.
I don't know if the id will be the same to check for yourself via console and HARD SAVE before you do any commands.

TL;DR:
Use the console command setobjectivecomplete _SD_enslavement 61 1 to force the inspection to finish, check your own ids and save before using.
